### ISSUE TITLE
Youtube videos freeze when you open a second video

### ISSUE THREAD

**Issue by BubiBalboa** [grubeninspekteur/BetterRBTV#44]
If I'm watching a video on Youtube and open a new video in another tab the first video stops playing and both videos freeze. The only way to fix this is to close all videos but one and reload the page. This happens on Win10 64bit, FF 49.0.1, HMTL5-Video, no Flashplayer installed. I have multiple addons installed but it happens whether I run BetterRBTV  with or without my other addons active. The video also freezes when you activate the addon while the video is playing.

I hope this bug is reproducible and easily fixable for you. A quick workaround might be to have BetterRBTV only work on gaming.youtube.com (maybe?).

**Comment by grubeninspekteur** [comment]
Thanks for your report! Unfortunately, I was unable to reproduce the issue with Firefox 49.0.1. Though I certainly did not know that Firefox injects scripts into existing tabs when enabling an add-on. I also notice the script being injected on every navigation (twice), for reasons I have yet to figure out.

**Could you do the following:**
1. Open the console for the first video (right click into page -> Inspect Element -> tab "Console")
2. Perform one of the steps that cause the freeze

Tell me whether there is a large amount of "Looking for chat..." messages in the console. They might be aggregated and the number of messages indicated by a number after the line.

While we could restrict the script to only gaming.youtube.com (we actually did before), we would like it to work for YouTube Live also, since a number of viewers of the channel it was created for use this interface.

**Comment by BubiBalboa** [comment]
I tried to reproduce the behavior today and was unable to. I tried to make sure it wasn't just a fluke and restarted the browser multiple times with different add-ons active before opening the issue here. But today everything works fine. That's good, I guess, but I feel a little bad about alarming you over nothing. 

I have activated extension logging in FF and will post a log if it happens again. Thank you for your great work.
